packages-ts/gauntlet-terra/src/commands/ledgerKey.ts

```diff
@@ -0,0 +1,66 @@
+import { Key, SimplePublicKey } from '@terra-money/terra.js'
+import LedgerTerraConnector, {ERROR_CODE} from '@terra-money/ledger-terra-js'
+import TransportNodeHid from "@ledgerhq/hw-transport-node-hid"
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
+
+
+export class LedgerKey extends Key {
+    private path: Array<number>
+
+    private constructor(path: Array<number>) {
+        super()
+        this.path = path
+    }
+
+    public async initialize() {
+        const {ledgerConnector, terminateConnection} = await this.connectToLedger()
+
+        const response = await ledgerConnector.getPublicKey(this.path)
+        if (response.return_code !== ERROR_CODE.NoError) {
+            logger.error(`LedgerKey init failed: ${response.error_message}. Is Ledger unlocked and Terra app open?`)
+            throw new Error(response.error_message)
+        }
+
+        this.publicKey = new SimplePublicKey(Buffer.from(response.compressed_pk.data).toString('base64'))
+        await terminateConnection()
+    }
+
+    public static async create(path: string): Promise<LedgerKey> {
+        const pathArr = this.pathStringToArray(path)
+        const ledgerKey = new LedgerKey(pathArr)
+        await ledgerKey.initialize()
+
+        return ledgerKey
+    }
+
+    private static pathStringToArray(path: string): Array<number> {
```

Comment:
Users should be able to enter the path on the standard format (`44'/60'/0'/0'/0`). Can we support that?

Reply:
done